grammar/TypeQL.g4

```diff
@@ -113,14 +113,16 @@ variable_concept      :   VAR_  IS  VAR_  ;
 variable_type         :   type_any    type_constraint ( ',' type_constraint )*  ;
 type_constraint       :   ABSTRACT
                       |   SUB_        type_any
-                      |   OWNS        type         ( AS type )? ( IS_KEY )?
+                      |   OWNS        type         ( AS type )?     annotations_owns
                       |   RELATES     type         ( AS type )?
                       |   PLAYS       type_scoped  ( AS type )?
                       |   VALUE       value_type
                       |   REGEX       STRING_
                       |   TYPE        label_any
                       ;
 
+annotations_owns      :   ( ANNOTATION_KEY )?   ( ANNOTATION_UNIQUE )?          ;
```

Comment:
@krishnangovindraj  raised good point about order indendence:

```
OWNS type ( AS type )? (annotation_owns*)

annotation_owns = ANNOTATION_KEY | ANNOTATION_UNIQUE | ANNOTATION_CARD ...
```

note that now each one is optional independently and the rule is written in singular form.